os/arch/arm/src/armv7-m/mpu.h

```diff
@@ -72,20 +72,6 @@
  * Pre-processor Definitions
  ************************************************************************************/
 
-/*
```

Comment:
@manoharahk @kishore-sn This file has reverts of what Girish did. I guess NXP just updates this as Nuttx's. Could you review this?

Reply:
@sunghan-chang 
As I know, the MPU proposal by Girish to use the 8 regions as below was not accepted due to various reasons. So, I'm not sure how this code was included. However, in our new proposal of MPU protection for binary separation, we plan to use the MPU regions in a different way. So, I think it is OK to take this Nuttx implementation for now. We will include the new MPU configuration as part of the binary separation activity.

Reply:
@kishore-sn Thank you for update.